Sources/Extensions/MockServer+Async.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  Created by Oliver Jones on 30/11/2022.
+//  Copyright © 2022 Oliver Jones. All rights reserved.
+//
+//  Permission to use, copy, modify, and/or distribute this software for any
+//  purpose with or without fee is hereby granted, provided that the above
+//  copyright notice and this permission notice appear in all copies.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+//  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+//  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY
+//  SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+//  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+//  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR
+//  IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+//
+
+import Foundation
+
+#if os(Linux)
+import PactSwiftMockServerLinux
+#elseif compiler(>=5.5)
+@_implementationOnly import PactSwiftMockServer
+#else
+import PactSwiftMockServer
+#endif
+
+extension MockServer {
+	
+	/// Spins up a mock server with expected interactions defined in the provided Pact
+	///
+	/// - Parameters:
+	///   - pact: The pact contract
+	///   - protocol: HTTP protocol
+	///
+	@available(macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0, *)
+	@discardableResult
+	func setup(pact: Data, protocol: PactSwiftMockServer.TransferProtocol = .standard) async throws -> Int {
```

Comment:
I think this is failing to build on Linux because the `PactSwiftMockServer` is known as `PactSwiftMockServerLinux` over there. Perhaps some special typealiasing is in order?

```swift
	#if os(Linux)
	private typealias MockTransferProtocol = PactSwiftMockServerLinux.TransferProtocol
	#else
	private typealias MockTransferProtocol = PactSwiftMockServer.TransferProtocol
	#endif
```

Reply:
Yeah. I've pushed some commits to actually remove these async functions from the Linux build. I don't have a Linux install to test this on so it is probably best to just make these Mac/iOS only additions to the API.